client/src/components/RegistrationContainer.js

```diff
@@ -22,37 +25,47 @@ export default function RegistrationContainer() {
     };
 
     //Registration action
-    const register = (e) => {
+    const register = async (e) => {
         //prevent page from refreshing
         e.preventDefault();
 
         //Registration code (send info to Back-end database)
 
         //Transform the user inputs to the user data
+        //The userId will be generated on the server.
         let userData = {
-            userId: Math.floor(Math.random() * 1000),
+            userId: 0,
             name: registrationState.nameInput,
             username: registrationState.userNameInput,
             email: registrationState.emailInput,
             password: registrationState.passwordInput,
             age: registrationState.ageInput,
-            lastLogin: new Date(),
+            lastLogin: "00-00-0000", 
             isLoggedIn: false,
             entries: []
         };
```

Comment:
```name```, ```username``` and ```password``` should be limited and validated/sanitized to avoid having to deal with someone sending megabytes through the form. Yup is an excellent tool for that -> https://www.npmjs.com/package/